tools/run.sh

```diff
@@ -3,7 +3,7 @@ set -ex
 
 PYTHON="$HOME"/miniconda3/bin/python
 DANDISETS_PATH=/mnt/backup/dandi/dandisets-healthstatus/dandisets
-MOUNT_PATH=/mnt/backup/dandi/dandisets-healthstatus/dandisets-fuse
+MOUNT_PATH=/tmp/dandisets-fuse
```

Comment:
let's do it under some safer user-specific location, e.g. `/var/run/user/$UID/`.

Reply:
Why?  `/mnt/backup/dandi/dandisets-healthstatus/dandisets-fuse` is the path we've been using on drogon for FUSE-mounting the Dandisets.  Also, `tools/run.sh` is just for generating the healthstatus reports; the shell script for generating the benchmarks is unfinished and hasn't been committed yet.

Reply:
It is a general security concern/practice to avoid: As `/tmp` is allowed to be written by anyone, it is not recommended to rely on pre-defined paths under there, since attacker potentially cause a problem by establishing a symlink to the target file/location "on your behalf".  `/mnt/backup/dandi/dandisets-healthstatus/` is allowed to be written only by dedicated user(s).

Reply:
You'll need to update the allowed `sudo mount` commands to operate on `/mnt/backup/dandi/dandisets-healthstatus/dandisets-fuse` instead of `/tmp/dandisets-fuse`.

Reply:
ok, on drogon I made following changes under `/etc`

```patch
diff --git a/davfs2/davfs2.conf b/davfs2/davfs2.conf
index 2f0053e..e6034e3 100644
--- a/davfs2/davfs2.conf
+++ b/davfs2/davfs2.conf
@@ -46,7 +46,7 @@
 # ignore_dav_header 0
 # use_compression 0
 # min_propset     0
-# follow_redirect 0
+follow_redirect 1
 # sharepoint_href_bug 0
 # server_charset
 # connect_timeout 10                # seconds
diff --git a/sudoers.d/fuse b/sudoers.d/fuse
new file mode 100644
index 0000000..fa21fd9
--- /dev/null
+++ b/sudoers.d/fuse
@@ -0,0 +1,4 @@
+jwodder ALL=(ALL:ALL) NOPASSWD: /usr/bin/umount /tmp/dandisets-fuse
+jwodder ALL=(ALL:ALL) NOPASSWD: /usr/bin/mount -t davfs https\://webdav.dandiarchive.org /tmp/dandisets-fuse
+jwodder ALL=(ALL:ALL) NOPASSWD: /usr/bin/umount /mnt/backup/dandi/dandisets-healthstatus/dandisets-fuse
+jwodder ALL=(ALL:ALL) NOPASSWD: /usr/bin/mount -t davfs https\://webdav.dandiarchive.org /mnt/backup/dandi/dandisets-healthstatus/dandisets-fuse
```